aliasMultinomial.lua

```diff
@@ -0,0 +1,13 @@
+local AM = torch.class('torch.AliasMultinomial') 
```

Comment:
I should have clarified that I wanted this function to support the formats `torch.multinomial` supported. This included `torch.multinomial(p, n)`. 

@amartya18x If possible we should be doing both. If not possible, stick with the current API for the function.